designer/deployment-manager-api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/InconsistentStateDetector.scala

```diff
@@ -1,14 +1,13 @@
-package pl.touk.nussknacker.ui.process.deployment
+package pl.touk.nussknacker.engine.api.deployment
```

Comment:
I would add it some nested package like `iconsistent` or `utils` to separate it from "root" of api